src/cli.js

```diff
@@ -2,7 +2,7 @@ import path from 'path';
 
 import argv from 'yargs';
 
-import { version } from 'json!../package';
```

Comment:
I've never seen this syntax. Not sure how/why it worked before, but it doesn't anymore...